src/Composer.js

```diff
@@ -1,10 +1,13 @@
+/* eslint no-use-before-define: ["error", { "variables": false }] */
+
 import PropTypes from 'prop-types';
 import React from 'react';
-import {
-  Platform,
-  StyleSheet,
-  TextInput,
-} from 'react-native';
+import { Platform, StyleSheet, TextInput } from 'react-native';
+
+import Constants from './Constants';
```

Comment:
Nit: also singular `Constant`

Reply:
Also, typically you can destructure in the import instead of below:

```js
import { MIN_COMPOSER_HEIGHT, DEFAULT_PLACEHOLDER } from './Constant';
```